docs/html/ux-research-design/index.md

```diff
@@ -0,0 +1,7 @@
+# UX Research & Design
```

Comment:
Please make sure to also update `docs/html/index.md`. It currently references `ux_research_design` which should be updated to `ux_research_design/index`.

Further, this will need a "toctree" similar to that page, listing all the sub-documents, so that they show up in the sidebar.

Reply:
ok, thanks @pradyunsg, will do.
I wasn't clear if it should be `ux_research_design/index` or `ux-research-design/index`?
The folder is `ux-research-design` should I change it to `ux_research_design` ?

Reply:
Ah, I didn't notice.

Yea, changing the folder name to underscores would mean that we don't change the URL for these docs, preserving them at https://pip.pypa.io/en/stable/ux_research_design/.

Reply:
(editted the last comment extensively, in case someone's following via email)